### PR TITLE
Change main content area bg to white, keep area above tabs grey

### PR DESCRIPTION
**Pull request by chriswatt**
Dashboard, repo pages and organization have all have the been modified

See screenshot.

bg1

### CODE REVIEW

**Comment by rajder** [comment]
Nice!

**Comment by Unknwon** [comment]
Thanks your PR!

I figure it would be nice to post screenshots before changes as well for comparsion?

**Comment by chriswatt** [comment]
Before and after comparisions

### Repo
gogsba1

### Org
gogsba2

### Dashboard
gogsba3

**Comment by Unknwon** [comment]
Thanks.

What if the page does not have a header?

I think `ui subheader` might not be a good choice right now? Or maybe we could come up something that only change the `padding-top: 15px;` once without add any HTML?

**Comment by chriswatt** [comment]
I can achieve the same effect without removing the `padding-top: 15px` at all

gogssubdiv

However because these 2 elements are on the same level, I still believe a they need to be wrapped in a new element in order to make the background differ

    <div class="ui container">
    <div class="ui tabs container">

**Comment by Unknwon** [comment]
Good point, but these two elements only exists in repo-related views. To make a general solution, I still suggest do not wrapped inside another `div` zone.

**Comment by Unknwon** [comment]
You can argue with me until one of us is convinced.

**Comment by chriswatt** [comment]
Ok I'm back to argue. 

This is a new attempt using the method I mentioned above. I don't touch the padding-top-15px of the page itself, instead modify the new wrapper to do the same thing. 

This change only affects the repo page as it's the only one with tabs thus is the main to benefit from it.

For a more general solution that would affect other pages with a header, I think the each page would need to be restructured in a similar fashion for this to work.

File: public/css/gogs.css [hunk 1]
```diff
@@ -2096,6 +2173,27 @@ footer .container .links > *:first-child {
   width: 15px;
   height: 15px;
 }
+#avatar-arrow:before,
```
Comment (Unknwon) [inline public/css/gogs.css]: What's going on with these lines?

**Comment by Unknwon** [comment]
Can you do `git reset --soft HEAD^7` and create a clean commit again.

Then use `git push --force` to your branch?

Changes loos very strange right now.

File: public/css/gogs.css (hunk 1 above)
Comment (chriswatt) [inline public/css/gogs.css]: Those are from my previous pr for the arrows on the comment form, gogs.css wasn't generated after you merged it

**Comment by chriswatt** [comment]
Yeah looking at the diff my branch does look a bit weird. I'll try fix

File: public/css/gogs.css (hunk 1 above)
Comment (Unknwon) [inline public/css/gogs.css]: I see.

**Comment by Unknwon** [comment]
Much better now!

**Comment by Unknwon** [comment]
Can you provide screenshots for an empty repo?

**Comment by chriswatt** [comment]
Sure ( ͡° ͜ʖ ͡°)

gogsemptyrepo

**Comment by Unknwon** [comment]
Thanks!

**Pull request merged**